lib/Conversion/TTNNToEmitC/TTNNToEmitC.cpp

```diff
@@ -93,6 +118,43 @@ class DefaultOpConversionPattern
   }
 };
 
+// MultiplyOp conversion pattern
+//
+// TODO(bug #623):
+// Convert all DPS-supported ttnn ops to this conversion pattern (nullopts added
+// for correct signature)
+//
+class MultiplyOpConversionPattern
+    : public TTNNToEmitCBaseOpConversionPattern<ttnn::MultiplyOp> {
+
+public:
+  MultiplyOpConversionPattern(const TypeConverter &typeConverter,
+                              MLIRContext *context, PatternBenefit benefit = 1)
+      : TTNNToEmitCBaseOpConversionPattern<ttnn::MultiplyOp>(typeConverter,
+                                                             context, benefit) {
+  }
+
+  LogicalResult
+  matchAndRewrite(ttnn::MultiplyOp srcOp, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+
+    llvm::SmallVector<Attribute, 5> attrs;
+    attrs.push_back(mlir::IntegerAttr::get(rewriter.getIndexType(), 0));
```

Comment:
Maybe just add a comment that emitc CallOpaqueOp will reference replaced op's operands via these indexes.